tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -477,29 +401,46 @@ public function userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles(s
 	 * @throws Exception|GuzzleException
 	 */
 	public function theseUsersHaveBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles(TableNode $table):void {
-		$originalSkeletonPath = $this->setSkeletonDirByType($this->getSmallestSkeletonDirName());
-		try {
-			$this->createTheseUsers(true, true, true, $table);
-		} finally {
-			// restore skeleton directory even if user creation failed
-			$this->setSkeletonDir($originalSkeletonPath);
-		}
+		$this->usersHaveBeenCreated($table);
 	}
 
 	/**
-	 * @Given /^these users have been created without skeleton files ?(and not initialized|):$/
-	 * expects a table of users with the heading
-	 * "|username|password|displayname|email|"
-	 * password, displayname & email are optional
+	 * @Given the user :byUser has created a new user using the Graph API with the following settings:
 	 *
+	 * @param string $byUser
 	 * @param TableNode $table
-	 * @param string $doNotInitialize
 	 *
 	 * @return void
+	 * @throws Exception|GuzzleException
+	 */
+	public function theAdministratorHasCreatedANewUserWithFollowingAttributes(string $byUser, TableNode $table): void {
```

Comment:
```suggestion
	public function theAdministratorHasCreatedANewUserWithFollowingSettings(string $byUser, TableNode $table): void {
```

Reply:
Would have been needed for upcoming PR but that's fine, we can change it again.